src/core/components/Paginate/index.js

```diff
@@ -33,6 +38,10 @@ export default class Paginate extends React.Component {
 
   visiblePages() {
     const { currentPage, showPages } = this.props;
+    if (!showPages) {
```

Comment:
Yeah, I figured it wasn't doing any harm and `showPages={false}` actually felt like a more expressive API but I didn't want to rule out `showPages={0}` for those that know the underpinnings more.

I wrote tests for both.